### ISSUE TITLE
Question about reproducing.

### ISSUE THREAD

**Issue by mrazhou** [wp03052/MATRN#8]
Thanks for your great work.

Can you tell me how long the model needs to be trained under the configuration of  4 NVIDIA GeForce RTX 3090GPUs  to converge to the results in the paper? If it is convenient, could you provide your training logs? 

I'm in the process of reproducing it now, but I found that the loss became jittery after a period of training, I don't know if I configured it wrong or if it's inherently so, slowly converging to the result in a long period of jitters. So I hope the author will provide a training log, if possible（thanks a lot).

Thank you very much!

**Comment by byeonghu-na** [comment]
Thank your for your interest.

We trained the model for around a week using 4 NVIDIA GeForce RTX 3090 GPUs for 10 epochs.
We attached the plot of test accuracy along iterations.
The performance increases significantly at around 250,000 step in which learning rate is decayed. Therefore, the learning rate decay technique improves the performance.

![training_cur](https://user-images.githubusercontent.com/43334491/160315407-66cff5b7-ece2-450a-9c13-ef4d56d4eb97.png)

Thank you.
